.github/ISSUE_TEMPLATE/but_report_template.md

```diff
@@ -0,0 +1,45 @@
+---
+name: 'Bug Report'
+about: 'Report a bug in ToastUI'
+labels: bug, help wanted
+---
+
+## Pre-requisites:
+- [ ] Yes, I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) looking for what I needed
+- [ ] No, I did not find what I was looking for
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+
+<!--- Tell us what should happen -->
+
+## Current Behavior
+
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas as to the implementation of the addition or change -->
+
+## Steps to Reproduce (for bugs)
+
+<!--- Provide a set of steps to -->
+<!--- reproduce this bug. Include code, configuration or screenshots to reproduce, if relevant -->
+
+
+## Context
+
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context (e.g. links to configuration settings, -->
+<!--- stack trace or log data) helps us come up with a solution that is most useful in the real world -->
```

Comment:
You should add period after `addition or change`, `if relevant` and `in the real world` comments as those are block comments. It's just my preference.

```suggestion
## Possible Solution

<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas as to the implementation of the addition or change. -->

## Steps to Reproduce (for bugs)

<!--- Provide a set of steps to -->
<!--- reproduce this bug. Include code, configuration or screenshots to reproduce, if relevant. -->

## Context

<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context (e.g. links to configuration settings, -->
<!--- stack trace or log data) helps us come up with a solution that is most useful in the real world. -->
```

Reply:
Fair enough. fixed in latest commits